src/Microsoft.NetCore.Analyzers/Core/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -1128,4 +1128,22 @@
   <data name="SetHttpOnlyForHttpCookieMessage" xml:space="preserve">
     <value>HttpCookie.HttpOnly is set to false or not set at all when using an HttpCookie. Ensure security sensitive cookies are marked as HttpOnly to prevent malicious scripts from stealing the cookies</value>
   </data>
+  <data name="DeprecatedSslProtocolsDescription" xml:space="preserve">
+    <value>Older protocol versions of Transport Layer Security (TLS) are less secure than TLS 1.2 and TLS 1.3, and are more likely to have new vulnerabilities. Avoid older protocol versions to minimize risk.</value>
+  </data>
+  <data name="DeprecatedSslProtocolsMessage" xml:space="preserve">
+    <value>Hardcoded use of deprecated Transport Layer Security protocol version '{0}'</value>
```

Comment:
This seems quite terse. I see that there's a description, but maybe a phrasing like <value>Transport Layer Security protocol version '{0}' is hardcoded. This version is deprecated as it is insecure. Using a more modern version and refraining from hardcoding a version are preferable.</value> would be better.

Reply:
How abouts:
Transport Layer Security protocol version '{0}' is deprecated.  Use 'None' to let the Operation System choose a version.